# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/html/HTMLElementAccessibles.cpp
# Commit: 9309568a622d
# Full Hash: 9309568a622de3c0e7fe7da366808ca17d681931
# Author: Morgan Rae Reschenberg <mreschenberg@mozilla.com>
# Date: 2022-07-27 09:37:31
# Regressor Bug: 1774043
# File Overlap Count: 1
# Description:
#   Bug 1774043: [Part 1] Add cache domains, caching logic for LABEL_FOR and LABELLED_BY relations r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152100
# ==============================================================================

diff -r b4fbf2bd8c4a -r 9309568a622d accessible/html/HTMLElementAccessibles.cpp
--- a/accessible/html/HTMLElementAccessibles.cpp	Tue Jul 26 21:31:57 2022 +0000
+++ b/accessible/html/HTMLElementAccessibles.cpp	Tue Jul 26 21:35:38 2022 +0000
@@ -66,6 +66,7 @@
                                                aModType, aOldValue, aOldState);
 
   if (aAttribute == nsGkAtoms::_for) {
+    mDoc->QueueCacheUpdate(this, CacheDomain::Relations);
     SendCache(CacheDomain::Actions, CacheUpdateType::Update);
   }
 }
